packages/actions/src/Concerns/HasDropdown.php

```diff
@@ -13,8 +13,11 @@ trait HasDropdown
 
     protected string | Closure | null $dropdownMaxHeight = null;
 
+    protected int | Closure | null $dropdownOffset = 8;
```

Comment:
Can this be null by default?

Reply:
It could, I kept it here so that the default would be clearly visible in User land without having to dig to the underline blade component.

What do you think? Make it null here too and leave the default in the blade component?